docs/UserGuide.md

```diff
@@ -1,134 +1,301 @@
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
-3. Double-click the file to start the app. The GUI should appear in a few seconds. 
-   > <img src="images/Ui.png" width="600">
+2. Download the latest `simply.jar` from the [releases](../../../releases) tab.
+3. Copy the file to the folder you want to use as the home folder for your Simply.
+4. Double-click the file to start the app. The GUI should appear in a few seconds. 
+  
+ <img src="images/Startup3.PNG" width="600"><br>
 
-4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
+5. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
    e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window. 
-5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` : 
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
+
 6. Refer to the [Features](#features) section below for details of each command.<br>
 
 
 ## Features
 
-> **Command Format**
-> * Words in `UPPER_CASE` are the parameters.
-> * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
-> * The order of parameters is fixed.
+###1 Add Command
+There are three variations to the add command. You are able to choose a task to be categorized under events, deadlines or to-dos. To differentiate the formatting for these commands, refer to the section below.
+
+####1.1 Add an event
+
+>Format : **Add** [&lt;event description&gt;; &lt;date(DDMMYY)&gt;; &lt;start time&gt;; &lt;end time&gt;]
+
+Example: Add [Sam's birthday party; 20/12/16 ; 9pm]
+
+>Note: The start time and end time is optional when adding an event.
+Important: The square brackets are compulsory when adding an event.
+
+####1.2 Add a deadline
+
+>Format: **Add** &lt;deadline description&gt;; &lt;date(DDMMYY)&gt;; &lt;end time&gt;
+
+Example: Add complete report; 120916; 1900
+
+>Important: The date and end time are compulsory when adding an deadline
+
+####1.3 Add a to-do
+
+>Format: **Add** &lt;to-do description&gt;
+
+Example: Add go swimming
+
+>Note: To do tasks do not have a date or time.
+
+>***Note: To add a tag, simply add a # followed by the tag name at the back of the command 
+To add a priority, simply add a ! at the back of the command.***
+
+####1.4 Add a tag / multiple tags
+
+>Format: **Add** &lt;index&gt; #tag1 #tag2
+
+Example: Add E1 #win #victor #finally  
+
+###2 Edit a task
+
+The multiple edit task functions allow you to update specific task details.
+
+####2.1 Edit all task details in one go
+
+>Format: **Edit** &lt;index&gt; [&lt;event description&gt;; &lt;date&gt;; &lt;start time&gt;; &lt;end time&gt;; &lt;tags&gt;; !] 
```

Comment:
it is for marking priorities so it should be added after the tags and separated by a ;